modules/ocs2/src/main/scala/gem/ocs2/pio/PioDecoder.scala

```diff
@@ -0,0 +1,39 @@
+package gem.ocs2.pio
+
+import PioError._
+
+import java.time.Instant
+
+import scala.xml.Node
+import scalaz.Scalaz._
+import scalaz._
+
+/** Typeclass for decoders of PIO-ish XML exported from an OCS2 database.
+  */
+trait PioDecoder[A] { outer =>
+  def decode(n: Node): PioError \/ A
+}
```

Comment:
`outer` isn't used here. This is isomorphic to `Kleisli[PioError \/ ?, Node, A]` which is a monad and a bunch of other things, so we might want to add at least `Functor` here.